client/cli/ok.py

```diff
@@ -189,7 +189,7 @@ def parse_input(command_input=None):
     server.add_argument('--update', action='store_true',
                         help="update ok and exit")
     # used in faded-parsons-frontend repo
-    server.add_argument('--fpp', action='store_true', 
+    server.add_argument('--parsons', action='store_true', 
                         help="run faded parsons problems in browser")  
```

Comment:
everything else looks okay. do you want this to be "run parsons problems," or do you want to leave the "faded" in since it's not an abbreviation here?

Reply:
Hm, guess I might as well be less specific here, though I'm not so worried when the whole thing is written out. Removed